sdk/src/main/scala/com/horizen/block/SidechainBlockBase.scala

```diff
@@ -135,6 +135,8 @@ abstract class SidechainBlockBase[TX <: Transaction, H <: SidechainBlockHeaderBa
       throw new InvalidSidechainBlockDataException(s"${getClass.getSimpleName} $id sidechain transactions amount exceeds the limit.")
 
     // Check Block size
+    // TODO in account model this should not be checked.
+    //      An option would be to handle it like the check above on number of TXs
```

Comment:
this todo can be removed, since in other PR it was implemented that `blockExceedsSizeLimit` always returns `false` for Account block

Reply:
Done